cmd/rook/rook/rook.go

```diff
@@ -199,6 +201,11 @@ func GetOperatorServiceAccount(clientset kubernetes.Interface) string {
 	// Getting the info of the operator pod
 	pod, err := k8sutil.GetRunningPod(clientset)
 	TerminateOnError(err, "failed to get pod")
+	resource := pod.Spec.Containers[0].Resources
```

Comment:
IMO, this could be a separate method as this method is only for getting the operator service account.